controllers/BookController.js

```diff
@@ -41,14 +38,18 @@ const getAllBooks = (req, res) => {
 };
 
 const getBookDetail = (req, res) => {
-    let { id } = req.params;
-
-    let sql = 'SELECT * FROM books LEFT JOIN category ON books.category_id = category.id WHERE books.id = ?;';
-    conn.query(sql, id, (err, results) => {
-        if (err) {
-            console.log(err);
-            return res.status(StatusCodes.BAD_REQUEST).end();
-        }
+    let { user_id } = req.body;
+    let { book_id } = req.params;
+
+    let sql = `SELECT *,
+                    (SELECT count(*) FROM likes WHERE books.id = book_id )AS likes,
+                    (SELECT EXISTS (SELECT * FROM likes WHERE user_id=? AND book_id=?)) AS liked
```

Comment:
EXISTS를 SELECT로 감싸지 않아도 됩니다😊
```suggestion
                    EXISTS (SELECT * FROM likes WHERE user_id=? AND book_id=?) AS liked
```

Reply:
오! SELECT로 감싸지 않아도 정상 작동하는군요 감사합니다~!